src/tools/gophish_import.py

```diff
@@ -202,29 +203,30 @@ def build_campaigns(api, assessment):
                 break
             except Error as e:
                 if e.message == "SMTP name already in use":
-                    logging.warning(f"{e}. Finding previously loaded smtp to delete.")
+                    logging.warning("%s. Finding previously loaded smtp to delete.", e)
                     smtps = api.smtp.get()
                     logging.debug(
-                        f"Checking {len(smtps)} for previously imported smtp profiles to get ID"
+                        "Checking %i for previously imported smtp profiles to get ID",
+                        len(smtps),
                     )
                     for old_smtp in smtps:
                         if old_smtp.name == new_smtp.name:
-                            logging.debug(f"Deleting SMTP with ID {old_smtp.id}")
+                            logging.debug("Deleting SMTP with ID %i", old_smtp.id)
                             api.smtp.delete(old_smtp.id)
                             logging.info("Re-Loading new SMTP.")
                 else:
-                    logging.error(f"{e}\n")
+                    logging.error("%s", e.message)
```

Comment:
Why not use [`logging.exception()`](https://docs.python.org/3/library/logging.html#logging.Logger.exception) here instead of recreating the wheel?

Reply:
See 44e24edb9ac96eb42759bdea8a9229563811ca59

Reply:
Also been updated here 3ee423033fefa9873fd13190c0e6edb575cd0080